{{cookiecutter.github_repository}}/requirements/common.txt

```diff
@@ -12,6 +12,9 @@ argon2-cffi==16.1.0
 django-environ==0.4.0
 django-sites==0.9
 python-dotenv==0.6.0
+{%- if cookiecutter.add_django_auth_wall.lower() == 'y' %}
```

Comment:
can this be moved to production.txt, including the things in middleware as well?